### DISCUSSION TITLE
⭐️ Star exchange ⭐️

### DISCUSSION THREAD

**Discussion by 0x4248** [Schweinepriester/github-profile-achievements#720] [General]
Star my project called [Lab Space](https://github.com/awesomelewis2007/lab_space) and i will star yours back (Reply with your project).

**Comment by jd-apprentice** [comment]
Done! here is my project [dbn-tools](https://github.com/jd-apprentice/dbn-tools)

**Reply by 0x4248** [reply]
> Done! here is my project [dbn-tools](https://github.com/jd-apprentice/dbn-tools)

Done

**Reply by gelfik** [reply]
Done

**Reply by ArielMAJ** [reply]
@jd-apprentice I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by gelfik** [reply]
> @jd-apprentice I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
> Feel free to contribute as well, there are many somewhat "simple" issues.

Done
https://github.com/gelfik/gelfik

**Reply by ArielMAJ** [reply]
> > @jd-apprentice I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
> > Feel free to contribute as well, there are many somewhat "simple" issues.
> 
> Done https://github.com/gelfik/gelfik

Ty! Already starred you!

**Reply by ghost** [reply]
@ArielMAJ @gelfik @awesomelewis2007 @jd-apprentice 
all done, star back :  https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Reply by ArielMAJ** [reply]
@darkhackkr done. star me back https://github.com/ArielMAJ/test.me

**Reply by jd-apprentice** [reply]
> @ArielMAJ @gelfik @awesomelewis2007 @jd-apprentice all done, star back : https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

Done!

**Reply by ArielMAJ** [reply]
@darkhackkr , @jd-apprentice , I can't find your stars

**Reply by Rantoryu** [reply]
@jd-apprentice Done, please if You have some time star my repo: https://github.com/Rantoryu/Personal_Website

**Reply by michalm57** [reply]
Done, Please star my repo:
https://github.com/michalm57/currency_exchange

**Comment by gelfik** [comment]
Done

https://github.com/gelfik/gelfik

**Reply by ghost** [reply]
done, star back :  https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Reply by Veercodeprog** [reply]
https://github.com/Veercodeprog/full-stack-assignment , please both i starred you

**Reply by michalm57** [reply]
Done, Please star my repo:
https://github.com/michalm57/currency_exchange

**Comment by Veercodeprog** [comment]
You can take part in ongoing [coding problem](https://github.com/Veercodeprog/Data-Structures-implementation/discussions/categories/coding-discussion-share-your-solution-with-best-time-complexity-approach) discussion.

A daily problem will be posted, and the answer with the most efficient time complexity submitted first will be selected as the accepted solution.

**Reply by ArielMAJ** [reply]
Ok

**Reply by geamdev** [reply]
ok

**Reply by michalm57** [reply]
Done, Please star my repo:
https://github.com/michalm57/currency_exchange

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Reply by michalm57** [reply]
Done, Please star my repo:
https://github.com/michalm57/currency_exchange

**Comment by vinograd0v** [comment]


**Reply by ArielMAJ** [reply]
@mmanosalva I starred you back!

Congratz on your Starstruck badge!

**Reply by ghost** [reply]
@ArielMAJ done, star back :  https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Reply by ArielMAJ** [reply]
@darkhackkr i have starred you. Please star me back as well https://github.com/ArielMAJ/test.me/stargazers

**Reply by michalm57** [reply]
Done, Please star my repo:
https://github.com/michalm57/currency_exchange

**Comment by ghost** [comment]
done, star back :  https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Reply by ArielMAJ** [reply]
I still haven't received your star, so I'll be removing mine until you star me alright? Lmk when you do.
https://github.com/ArielMAJ/test.me/stargazers

**Reply by michalm57** [reply]
Done, Please star my repo:
https://github.com/michalm57/currency_exchange

**Comment by YuZhangWang** [comment]
done 

[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)

**Reply by michalm57** [reply]
Done, Please star my repo:
https://github.com/michalm57/currency_exchange

**Comment by Rantoryu** [comment]
@awesomelewis2007 
Done, please star my repo: https://github.com/Rantoryu/Personal_Website
I will star back anyone in the comment to this!

**Reply by michalm57** [reply]
Done, Please star my repo:
https://github.com/michalm57/currency_exchange

**Comment by michalm57** [comment]
Hi, please add star to my repo:
https://github.com/michalm57/currency_exchange
I star back for everyone who give it to me :).
